.vscode/settings.json

```diff
@@ -1,7 +1,8 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
```

Comment:
functionally equivalent but my editor kept changing it automatically
boolean version was deprecated: https://code.visualstudio.com/updates/v1_85#_code-actions-on-save-and-auto